skdecide/hub/domain/up/up.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 import unified_planning as up
+from gymnasium.spaces import Box
```

Comment:
@HamdaHmida Please use only the scikit-decide version, i.e. BoxSpace, which is already imported a few lines below.

Reply:
@fteicht j'ai fait le changement demandé et j'ai met à jour la branche